README.md

```diff
@@ -142,10 +152,34 @@ version of the ebookmaker package with `pipenv install`:
 pipenv install "ebookmaker==0.9.2"
 ```
 
+Get the relevant version of the tidy.conf file into the dist directory.
+You may need to update the version number in the script first:
+
+```
+./get_tidy_conf.sh
+```
+
 Then re-apply the patch to enable building with pyinstaller and build
 the new binary as instructed above. Depending on the changes within
 ebookmaker this may require updating the patch and/or the list of hidden
 imports passed into pyinstaller.
 
 Be sure and check in the updated `Pipfile` and `Pipfile.lock` along with
 any other changes before tagging and releasing a new binary.
+
+## Troubleshooting
+
+If ebookmaker crashes and the traceback refers to imported packages, it
+may be that one of the packages has been updated, requiring an updated
+version of pyinstaller:
+```
+pipenv install "pyinstaller==4.7"
```

Comment:
This implies that installing version 4.7 this will solve the problem. Perhaps this might be clearer:
```
If ebookmaker crashes and the traceback refers to imported packages, it
may be that one of the packages has been updated, requiring an updated
version of pyinstaller. Try upgrading to the latest version of PyInstaller
by finding the [latest released version](https://github.com/pyinstaller/pyinstaller/releases)
and upgrading to it with:
` ` `
pipenv install "pyinstaller==$LATEST_RELEASE"
` ` `
```

Reply:
And I just realized your example mirrors the instructions for updating ebookmaker (which I undoubtably wrote) so this already presumes some knowledge of updating to new releases, so maybe just ignore the above suggestion 😁 .